docs/Projects/LLM-Benchmarks/index.md

```diff
@@ -0,0 +1,27 @@
+# LLM Benchmarks
+
+This document contains the design decisions behind the LLM Benchmarks the architectural diagrams and the technical
+solution can be found [here](https://github.com/MinBZK/llm-benchmark).
+
+The requirements of the solution are:
+
+- People should be able to run the LLM Benchmark system locally
+    - When the BIO does not allow the model to leave the digital organizational boundary
+    - Locally we have to get the hash of the model as a version stamp
+    - Most likely this will be a containerized version of the solution
+- There should be an online version hosted by the ai-validation team for easy access and to promote transparency
+    - This should be scalable as downloading models and verifying them could take a long time especially when supplying
+    many people
+- The preferable entrance would be that people have their model versioned through version control and published via
+the [Dutch Algorithm Register](https://algoritmes.overheid.nl/nl)
+    - In this way people would only need to provide the link to the model on the register and our solution would do the
+    rest, the results of the benchmarks would also be posted on the Algorithm Register
+- The LLM Benchmarks system does not store any data long-term, the benchmark results and models are stored at the
```

Comment:
This requirement is a question i think. We might want to consided storing the results and add authentication & Authroization for individuals to access the reports generated.

Reply:
True!